doc/source/api.rst

```diff
@@ -24,9 +24,11 @@ PyAnsys packages API reference
         :gutter: 3 3 4 4
 
         {% for project, metadata in projects['projects'].items() %}
+          {% if 'other_tools' not in metadata or metadata['other_tools'] != True %}

```

Comment:
https://github.com/ansys/pyansys/pull/857/files#r1967196093 @jorgepiloto as mentioned here, i used the other_tools tags here.